content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -61,6 +61,21 @@ localization:
        <../accounting/customer_invoices/electronic_invoicing>`, based on the Technical
        documentation published by the SRI. The authorized documents are: Invoices, Credit Notes,
        Debit Notes, Withholdings and Purchase liquidations.
+   * - :guilabel:`Ecuadorian Accounting Reports`
+     - `l10n_ec_reports`
+     - Includes all the technical and functional requirements to generate forms 103 and 104.
+   * - :guilabel:`Ecuadorian ATS Report`
+     - `l10n_ec_reports_ats`
+     - Includes all the technical and functional requirements to generate ATS report XML file ready
+       to be uploaded to the *DIMM Formularios*.
+   * - :guilabel:`Ecuadorian Website`
+     - `l10n_ec_website_sale`
+     - Includes all the technical and functional requirements to generate automatic electronic
+       invoices from a Website sale.
+   * - :guilabel:`Ecuadorian Point of Sale`
+     - `l10n_ec_edi_pos`
+     - Includes all the technical and functional requirements to generate automatic electronic
+       invoices from a PoS sale.
```

Comment:
I don't see this one in 17.0 but I do see it in 17.2, am I missing something or is this something that should only be added from 17.2?

Reply:
@auva-odoo This module was just released this week in 17.2 and master. It will be backported next week to 17.0. Thanks for the review! @samueljlieber